README.md

```diff
@@ -7,22 +7,301 @@ Serlio requires so-called rule packages (RPK) as input, which are authored in Ci
 Serlio is well suited for managing the procedural generation of architectural 3D content in digital sets. However, Serlio is restricted to the procedural generation of single buildings / objects. Serlio does not include the city layouting and street network editing tools of CityEngine i.e. the rich CityEngine toolset to design a city from scratch (or based on geographic data) is still needed.
 
 
-## Documentation
+## Quick Start
+1. Use CityEngine to [export a Rule Package](https://doc.arcgis.com/en/cityengine/latest/help/help-rule-package.htm) (RPK).
+1. Acquire the Serlio binaries, either: 
+    * ... download the [desired release](https://github.com/Esri/serlio/releases)
+    (on linux you still need to manually [install the local build](#linux-2))
+    * ... or [build](#build-instructions) them yourself, which will result in an `install` directory
+1. Start Maya (Note: you may want to start maya from a shell to see the Serlio log output).
+1. Open the plugin manager: "Windows" > "Settings/Preferences" > "Plug-in Manager"
+1. Enable `serlio.so` / `serlio.mll`
+1. The plugin should load and a new menu item `Serlio` should appear in Maya.
+1. In Maya, select a mesh and use the `Serlio` menu to assign the RPK. This will apply the selected rule to each face in the mesh.
+1. Make sure the geometry is still selected and go to the `serlio` tab in the Attribute editor. Here you can edit the rule parameters.
+1. To create materials, apply one of the two commands in the `Serlio` menu on the generated model. Please note the the two material systems are mutually exclusive at this point.
 
-* [Home Page with Examples](https://esri.github.io/cityengine/serlio)
-* [Installation and Quick Start](doc/usage.md)
-* [Building Serlio](doc/build.md)
-* [ChangeLog](doc/changelog.md)
+
+## Table of Contents
+
+* [User Manual](#user-manual)
+* [Examples](https://esri.github.io/cityengine/serlio)
+* [Developer Manual](#developer-manual)
+* [Release Notes](#release-notes)
+* [Known Limitations](#known-limitations)
+* [Licensing information](#licensing-information)
 
 External documentation:
 * [CityEngine Tutorials](https://doc.arcgis.com/en/cityengine/latest/tutorials/introduction-to-the-cityengine-tutorials.htm)
 * [CityEngine CGA Reference](https://doc.arcgis.com/en/cityengine/latest/cga/cityengine-cga-introduction.htm)
 * [CityEngine Manual](https://doc.arcgis.com/en/cityengine/latest/help/cityengine-help-intro.htm)
 * [CityEngine SDK API Reference](https://esri.github.io/cityengine-sdk/html/index.html)
 
+## User Manual
+
+### Installation
+
+#### Windows: Provided Binaries
+1. Download the MSI installer for your Maya version from the [desired release](https://github.com/Esri/serlio/releases).
+1. Run the MSI installer.
+1. Start Maya.
+1. You should now see the new menu item `Serlio`.
+
+#### Windows: Install local build
+See [below](#windows-2) in the developer documentation.
```

Comment:
Leave this away - developers will understand that anyways.

Reply:
removed.